cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -2151,6 +2151,9 @@ def test_logout(self):
         # Logout redirects.
         self._test_page("/logout", 302)
 
+    def test_accessibility(self):
```

Comment:
How about testing the case where the waffle switch is disabled too? http://waffle.readthedocs.io/en/v0.12.0/testing/automated.html has an example